meta_inspector.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.version       = MetaInspector::VERSION
 
   gem.add_dependency 'nokogiri', '~> 1.11.0'
-  gem.add_dependency 'faraday', '~> 1.1.0'
+  gem.add_dependency 'faraday', '~> 1'
```

Comment:
We prefer being more specific about the minimum version, instead of `~> 1` please use `~> 1.4.1`